test/CosmosDB/Microsoft.Azure.Workflows.ServiceProvider.Extensions.CosmosDB.Tests/stylecop.json

```diff
@@ -0,0 +1,14 @@
+﻿{
+  // ACTION REQUIRED: This file was automatically added to your project, but it
+  // will not take effect until additional steps are taken to enable it. See the
+  // following page for additional information:
+  //
+  // https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/EnableConfiguration.md
+
+  "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
```

Comment:
can we actually enable stylecop so that we start with a enforcing certain set of styling rules?

Reply:
enabled the stylecop